doc/examples/segment_extending_clustering_framework.py

```diff
@@ -7,47 +7,51 @@
 an adjacency threshold to perform clustering. There is a wide variety of metrics
 that could be used to cluster streamlines.
 
-The purpose of this tutorial is to show how to easily create new `Feature` and
-new `Metric` classes that can be used by QuickBundles.
+The purpose of this tutorial is to show how to easily create new ``Feature`` and
+new ``Metric`` classes that can be used by QuickBundles.
 
 .. _clustering-framework:
 
 Clustering framework
 ====================
-Dipy provides a simple, flexible and fast framework to do clustering of
+dipy_ provides a simple, flexible and fast framework to do clustering of
```

Comment:
I guess it's because the `link_names.inc` file was NOT included. Fixed in 412e9fb.